src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  let myList = list;
```

Comment:
The variable `myList` is initially assigned the value of `list`, but this assignment is immediately overwritten in line 6. Consider removing this line or using it correctly.